chroma-manager/tests/framework/integration/shared_storage_configuration/full_cluster/run_tests

```diff
@@ -13,7 +13,13 @@ CHROMA_DIR=${CHROMA_DIR:-"$PWD/intel-manager-for-lustre/"}
 
 eval $(python $CHROMA_DIR/chroma-manager/tests/utils/json_cfg2sh.py "$CLUSTER_CONFIG")
 
-TESTS=${TESTS:-"tests/integration/shared_storage_configuration/"}
+if [ -z "$TESTS" ]; then
+    TESTS="tests/integration/shared_storage_configuration/"
+else
+    # if the caller specified test(s), make sure the
+    # test_setup_zfs_targets_for_test() hackery is included
+    TESTS="tests/integration/shared_storage_configuration/test_aaa_zfs_setup.py:TestConfigureZfsTargets.test_setup_zfs_targets_for_test $TESTS"
```

Comment:
why don't you just include the file name, rather than the test specifically?

Reply:
Because I tend to error on the side of caution and be explicit so that future changes don't cause intended consequences.